src/packages/@teams/features/create-team/layout/CreateTeam.svelte

```diff
@@ -0,0 +1,236 @@
+<script lang="ts" context="module">
+  interface NewTeam {
```

Comment:
Is it possible to shift the interface in new file?

Reply:
resolved. moved to types